docs/tools/tikv-control.md

```diff
@@ -107,6 +107,23 @@ key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\00
 
 In this command, the key is also the escaped form of raw key.
 
+### Scan raw keys
+
+The `raw-scan` subcommand is similar to `scan`, except that it doesn't try to interpret the keys as MVCC keys but prints the raw keys directly.
+
+Use `--from` and `--to` options to specify the range to scan (unbounded by default). Use `--limit` to limit at most how
+many keys to print out (30 by default). Use `--cf` to specify which cf to scan (can be `default`, `write` or `lock`).
+
+```bash
+$ ./tikv-ctl --db /var/lib/tikv/db/ raw-scan --from 'zt' --limit 2 --cf default
+key: "zt\200\000\000\000\000\000\000\377\005_r\200\000\000\000\000\377\000\000\001\000\000\000\000\000\372\372b2,^\033\377\364", value: "\010\002\002\002%\010\004\002\010root\010\006\002\000\010\010\t\002\010\n\t\002\010\014\t\002\010\016\t\002\010\020\t\002\010\022\t\002\010\024\t\002\010\026\t\002\010\030\t\002\010\032\t\002\010\034\t\002\010\036\t\002\010 \t\002\010\"\t\002\010$\t\002\010&\t\002\010(\t\002\010*\t\002\010,\t\002\010.\t\002\0100\t\002\0102\t\002\0104\t\002"
+key: "zt\200\000\000\000\000\000\000\377\025_r\200\000\000\000\000\377\000\000\023\000\000\000\000\000\372\372b2,^\033\377\364", value: "\010\002\002&slow_query_log_file\010\004\002P/usr/local/mysql/data/localhost-slow.log"
+
+Total scanned keys: 2
+```
+
+There's a limitation: if you are trying to scan write cf, the keys you provide shouldn't be shorter than 8 bytes.
```

Comment:
What happens if the key is shorter?

Reply:
Write CF has a `FixedSuffixSliceTransform`. If a key that is shorter than 8 bytes is provided, it will cause overflow when calculating the length.
I forgot to check the length before giving the key to rocksdb.. I just added it

Reply:
Oh, sorry, I've got into a mistake.. `total_order_seek` should be set and then it's ok to seek a key shorter than 8 bytes.. I've fixed it

Reply:
Great!